### ISSUE TITLE
Model field rules are not defined , the last rule of ActiveForm::attributes is applied.

### ISSUE THREAD

**Issue by Choate** [unclead/yii2-multiple-input#108]
```php
class User extends ActiveRecord
{
    public function rules() {
        return [
            [['first_name', 'last_name', 'email'], 'require'],
        ];
    }
}

// table fields: address, zipcode
class UserAddress extends ActiveRecord {
    public function rules() {
        return [
            ['address', 'required'],
        ];
    }
}


// View
echo $form->field('first_name');
echo $form->field('last_name');
echo $form->field('email');
echo TabularInput::widget([
    'models'           => $userAddressModels,
    'form'             => $form,
    'columns' => [
        [
            'name' => 'address',
            'enableError' => true,
        ],
        [
            'name' => 'zipcode',
            'enableError' => true,
        ],
    ],
]);
```

**zipcode** field shows error is **email** cannot be blank.

Fixed Code

```php
BaseRenderer::prepareJsAttributes

    protected function prepareJsAttributes()
    {
        $attributes = [];
        foreach ($this->columns as $column) {
            $model = $column->getModel();
            $inputID = str_replace(['-0', '-0-'], '', $column->getElementId(0));
            if ($this->form instanceof ActiveForm && $model instanceof Model) {
                $field = $this->form->field($model, $column->name);
                foreach ($column->attributeOptions as $name => $value) {
                    if ($field->hasProperty($name)) {
                        $field->$name = $value;
                    }
                }
                $field->render('');
                $attributeOptions = array_pop($this->form->attributes);
                if (isset($attributeOptions['name']) && $attributeOptions['name'] == $column->name) {
                    $attributes[$inputID] = ArrayHelper::merge($attributeOptions, $column->attributeOptions);
                }
            } else {
                $attributes[$inputID] = $column->attributeOptions;
            }
        }

        return $attributes;
    }
```

**Comment by unclead** [comment]
Show me the full version of a view file, please. I want to reproduce the issue

**Comment by unclead** [comment]
closed via 03d8d457f2044e4c5c0d6c8b48be95f2a6214932
thank you for your contribution
